### PR TITLE
fix(consumers): Create topics automatically by default

### PR DESCRIPTION
**Pull request by tonyo**
Fixes the issue introduced by #20208, when some ingest topics are not created automatically.
Relevant for local development and onpremise.

### CODE REVIEW

**Review by BYK — APPROVED** [review]

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1874,6 +1874,9 @@ def get_sentry_sdk_config():
     KAFKA_INGEST_TRANSACTIONS: {"cluster": "default", "topic": KAFKA_INGEST_TRANSACTIONS},
 }
 
+# If True, consumers will create the topics if they don't exist
+KAFKA_CONSUMER_AUTO_CREATE_TOPICS = True
```
Comment (BYK) [inline src/sentry/conf/server.py:1878]: I think we should set this to `False` in getsentry?

File: src/sentry/utils/batching_kafka_consumer.py [hunk 2]
```diff
@@ -168,6 +169,41 @@ def __record_timing(self, metric, value, tags=None):
         sample_rate = self.__metrics_sample_rates.get(metric, settings.SENTRY_METRICS_SAMPLE_RATE)
         return self.__metrics.timing(metric, value, tags=tags, sample_rate=sample_rate)
 
+    def _wait_for_topics(self, admin_client, topics, timeout=10):
+        """
+        Make sure that the provided topics exist and have non-zero partitions in them.
+        """
+        for topic in topics:
```
Comment (BYK) [inline src/sentry/utils/batching_kafka_consumer.py:176]: I know this is only 3 topics right now but I cannot help but feeling we should be using a thread pool or something here.

File: src/sentry/conf/server.py (hunk 1 above)
Comment (tonyo) [reply src/sentry/conf/server.py:1878]: Totally https://github.com/getsentry/getsentry/pull/4480

File: src/sentry/utils/batching_kafka_consumer.py (hunk 2 above)
Comment (tonyo) [reply src/sentry/utils/batching_kafka_consumer.py:176]: Yes, but as you mentioned, the added complexity would probably not worth it right now.

**Pull request merged**